joblib/testing.py

```diff
@@ -46,14 +46,20 @@ def check_subprocess_call(cmd, timeout=5, stdout_regex=None,
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE,
                             stderr=subprocess.PIPE)
 
+    def terminate_process():
+        warnings.warn("Timeout running {}".format(cmd))
+        proc.terminate()
+
     def kill_process():
         warnings.warn("Timeout running {}".format(cmd))
         proc.kill()
```

Comment:
I wonder why codecov does not complain here while it does complain for lines 50-51...